InstagrammApp/InstagrammApp/ViewController.swift

```diff
@@ -7,13 +7,12 @@
 
 import UIKit
 
+/// Класс основного ViewController
 class ViewController: UIViewController {
```

Comment:
```suggestion
final class NameViewController: UIViewController {
```
нужно дать название каждому view controller, вместо Name подставь то, что подходит по контексту

Reply:
а еще тут final забыл

Reply:
этот файл просто неправильно удалил, вообще его не использую нигде
вместо move to trash удалил ссылку